src/Microsoft.Windows.CsWin32/Generator.cs

```diff
@@ -2828,7 +2828,7 @@ private StructDeclarationSyntax DeclareTypeDefStruct(TypeDefinition typeDef)
                 return this.DeclareTypeDefBOOLStruct(typeDef);
             }
 
-            bool isHandle = false;
+            bool isHandle = name.Identifier.ValueText == "HGDIOBJ";
```

Comment:
Why is this name special-cased? Temporary measure until metadata takes charge of this determination?

Reply:
Ah, expanded the resolved conversation and found your answer. 👍